<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_event_widget_window</class>
 <widget class="QWidget" name="add_event_widget_window">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>–î–æ–±–∞–≤–∏—Ç—å —Å–æ–±—ã—Ç–∏–µ</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(34,39,51,0.137);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="add_event_horizontalLayout_main">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <spacer name="add_event_horizontalSpacer_main">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="add_event_verticalLayout_menu">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QWidget" name="add_event_widget_title_btn" native="true">
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(22, 29, 49);</string>
        </property>
        <widget class="QLabel" name="add_event_textlbl_title">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>225</width>
           <height>26</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(208, 210, 214);</string>
         </property>
         <property name="text">
          <string>–î–æ–±–∞–≤–∏—Ç—å —Å–æ–±—ã—Ç–∏–µ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="add_event_btn_exit">
         <property name="geometry">
          <rect>
           <x>324</x>
           <y>13</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
color: rgb(208, 210, 214);
background-color:rgba(0,0,0,0);
}
QPushButton:hover {
color: rgb(234, 84, 85);
}
</string>
         </property>
         <property name="text">
          <string>üû©</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="add_event_widget_form" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>9999</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(40, 48, 70);</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>10</y>
           <width>410</width>
           <height>1021</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="add_event_verticalLayout_shape_alignment">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>24</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="add_event_textlbl_name">
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(185, 185, 195);</string>
            </property>
            <property name="text">
             <string>–ù–∞–∑–≤–∞–Ω–∏–µ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="add_event_lineEdit_name">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
background-color: #283046;
border: 1px solid rgb(122, 122, 122);
border-radius: 5px;
color: rgb(255, 255, 255);
}
QLineEdit:hover {
	background-color: rgb(40, 48, 70); 
	border: 1px solid rgb(95, 85, 200);
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="add_event_textlbl_category">
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-top:10px;
color: rgb(185, 185, 195);</string>
            </property>
            <property name="text">
             <string>–ö–∞—Ç–µ–≥–æ—Ä–∏—è</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="add_event_comboBox_category">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto-Light</family>
              <pointsize>12</pointsize>
              <weight>3</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
font: 25 12pt &quot;Roboto-Light&quot;;
background-color: #283046;
border: 1px solid rgb(122, 122, 122);
border-radius: 5px;
color: rgb(255, 255, 255);
}
QComboBox:hover {
	background-color: rgb(40, 48, 70); 
	border: 1px solid rgb(95, 85, 200);
}
QComboBox QAbstractItemView {
background-color: rgb(22, 29, 49);
border: 2px solid #7666F8;
color: rgb(203, 203, 203);
selection-background-color: rgb(94, 80, 238);
selection-color: rgb(0, 0, 0);
}</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="add_event_textlbl_time">
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-top:10px;
color: rgb(185, 185, 195);</string>
            </property>
            <property name="text">
             <string>–ü–æ—Ç—Ä–∞—á–µ–Ω–æ –≤—Ä–µ–º–µ–Ω–∏</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="add_event_lineEdit_time">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
background-color: #283046;
border: 1px solid rgb(122, 122, 122);
border-radius: 5px;
color: rgb(255, 255, 255);
}
QLineEdit:hover {
	background-color: rgb(40, 48, 70); 
	border: 1px solid rgb(95, 85, 200);
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="add_event_textlbl_date">
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-top:10px;
color: rgb(185, 185, 195);</string>
            </property>
            <property name="text">
             <string>–î–∞—Ç–∞</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="add_event_dateEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QDateEdit {
background-color: #283046;
border: 1px solid rgb(122, 122, 122);
border-radius: 5px;
color: rgb(255, 255, 255);
}
QDateEdit:hover {
	background-color: rgb(40, 48, 70); 
	border: 1px solid rgb(95, 85, 200);
}</string>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="minimumDateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>1952</year>
              <month>9</month>
              <day>14</day>
             </datetime>
            </property>
            <property name="maximumDate">
             <date>
              <year>2100</year>
              <month>12</month>
              <day>31</day>
             </date>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2021</year>
              <month>5</month>
              <day>15</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="add_event_textlbl_comment">
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-top:10px;
color: rgb(185, 185, 195);</string>
            </property>
            <property name="text">
             <string>–ö–æ–º–º–µ–Ω—Ç–∞—Ä–∏–π</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="add_event_plaintextedit_comment">
            <property name="maximumSize">
             <size>
              <width>316</width>
              <height>126</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPlainTextEdit {
background-color: #283046;
border: 1px solid rgb(122, 122, 122);
border-radius: 5px;
color: rgb(255, 255, 255);
}
QPlainTextEdit:hover {
	background-color: rgb(40, 48, 70); 
	border: 1px solid rgb(95, 85, 200);
}</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="add_event_horizontalLayout_btns">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>40</number>
            </property>
            <item>
             <widget class="QPushButton" name="add_event_btn_add">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #7367F0;
border-radius: 5px;
border: 2px solid #7367F0;
color: rgb(255, 255, 255);
padding-top: 10 px; 
padding-right: 23 px;
padding-left: 23 px;
padding-bottom: 10 px;
text-align: center;
margin-right: 8px;
}
QPushButton:hover {
  background-color: rgb(40, 48, 70); 
  color: rgb(115, 103, 240);
  border: 2px solid rgb(95, 85, 200);
}</string>
              </property>
              <property name="text">
               <string>–î–æ–±–∞–≤–∏—Ç—å</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_event_btn_cancel">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>14</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: rgb(40, 48, 70); 
border-radius: 5px;
border: 2px solid rgb(234, 84, 85);
color: rgb(234, 84, 85);
padding-top: 10 px; 
padding-right: 23 px;
padding-left: 23 px;
padding-bottom: 10 px;
text-align: center;
margin-left: 8px;

}
QPushButton:hover {
background-color: rgb(234, 84, 85);
color: rgb(255, 255, 255);
border: 2px solid rgb(234, 84, 85);
}</string>
              </property>
              <property name="text">
               <string>–û—Ç–º–µ–Ω–∞</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="add_event_horizontalSpacer_btns">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="add_event_verticalSpacer_btns">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
